src/petals/utils/convert_block.py

```diff
@@ -77,15 +70,13 @@ def replace_8bit_linear(model: nn.Module, threshold=6.0):
     # Import bitsandbytes only when necessary, so Petals runs on platforms not supported by bitsandbytes
     import bitsandbytes as bnb
 
-    from petals.utils.linear8bitlt_patch import CustomLinear8bitLt
-
     for n, module in model.named_children():
         if len(list(module.children())) > 0:
             replace_8bit_linear(module, threshold)
 
         if isinstance(module, torch.nn.Linear) and n not in ["lm_head", "score"]:
             assert module.weight.device.type == "cpu", f"expected linear layers on CPU, got {module.weight.device}"
-            model._modules[n] = CustomLinear8bitLt(
+            model._modules[n] = bnb.nn.Linear8bitLt(
```

Comment:
Not strictly necessary, but it'd be good to get rid of all bitsandbytes-related code that got into upstream before merging this

Reply:
Done in #297.